src/applications/caregivers/components/PreSubmitInfo/index.jsx

```diff
@@ -227,13 +256,38 @@ const PreSubmitCheckboxGroup = ({ onSectionComplete, formData, showError }) => {
       </p>
 
       <div aria-live="polite">
-        <SubmitLoadingIndicator />
+        <SubmitLoadingIndicator submission={submission} />
       </div>
     </section>
   );
 };
 
+PreSubmitCheckboxGroup.propTypes = {
+  showError: PropTypes.bool.isRequired,
+  onSectionComplete: PropTypes.func.isRequired,
+  setFormData: PropTypes.func.isRequired,
+  formData: PropTypes.object.isRequired,
+  submission: PropTypes.shape({
+    hasAttemptedSubmit: PropTypes.bool,
+    errorMessage: PropTypes.oneOfType([PropTypes.bool, PropTypes.string]),
+    status: PropTypes.oneOfType([PropTypes.bool, PropTypes.string]),
+  }),
+};
+
+const mapStateToProps = state => {
+  return {
+    submission: state.form.submission,
+  };
+};
+
+const mapDispatchToProps = {
+  setFormData: setData,
+};
+
 export default {
   required: true,
-  CustomComponent: PreSubmitCheckboxGroup,
+  CustomComponent: connect(
+    mapStateToProps,
+    mapDispatchToProps,
+  )(PreSubmitCheckboxGroup),
```

Comment:
Refactored all redux connections removed the lower ones in favor of connecting the container and passing props down the chain.